seed/seed.coffee

```diff
@@ -1,22 +1,4337 @@
+  "aliases": ["airplane"],
+  "tags": ["flight"]
+}, {
+  "emoji": "💺",
+  "description": "seat",
+  "aliases": ["seat"],
+  "tags": []
+}, {
+  "emoji": "🚁",
+  "description": "helicopter",
+  "aliases": ["helicopter"],
+  "tags": []
+}, {
+  "emoji": "🚂",
+  "description": "steam locomotive",
+  "aliases": ["steam_locomotive"],
+  "tags": ["train"]
+}, {
+  "emoji": "🚊",
+  "description": "tram",
+  "aliases": ["tram"],
+  "tags": []
+}, {
+  "emoji": "🚉",
+  "description": "station",
+  "aliases": ["station"],
+  "tags": []
+}, {
+  "emoji": "🚞",
+  "description": "mountain railway",
+  "aliases": ["mountain_railway"],
+  "tags": []
+}, {
+  "emoji": "🚆",
+  "description": "train",
+  "aliases": ["train2"],
+  "tags": []
+}, {
+  "emoji": "🚄",
+  "description": "high-speed train",
+  "aliases": ["bullettrain_side"],
+  "tags": ["train"]
+}, {
+  "emoji": "🚅",
+  "description": "high-speed train with bullet nose",
+  "aliases": ["bullettrain_front"],
+  "tags": ["train"]
+}, {
+  "emoji": "🚈",
+  "description": "light rail",
+  "aliases": ["light_rail"],
+  "tags": []
+}, {
+  "emoji": "🚇",
+  "description": "metro",
+  "aliases": ["metro"],
+  "tags": []
+}, {
+  "emoji": "🚝",
+  "description": "monorail",
+  "aliases": ["monorail"],
+  "tags": []
+}, {
+  "emoji": "🚋",
+  "description": "tram car",
+  "aliases": ["train"],
+  "tags": []
+}, {
+  "emoji": "🚃",
+  "description": "railway car",
+  "aliases": ["railway_car"],
+  "tags": []
+}, {
+  "emoji": "🚎",
+  "description": "trolleybus",
+  "aliases": ["trolleybus"],
+  "tags": []
+}, {
+  "emoji": "🚌",
+  "description": "bus",
+  "aliases": ["bus"],
+  "tags": []
+}, {
+  "emoji": "🚍",
+  "description": "oncoming bus",
+  "aliases": ["oncoming_bus"],
+  "tags": []
+}, {
+  "emoji": "🚙",
+  "description": "recreational vehicle",
+  "aliases": ["blue_car"],
+  "tags": []
+}, {
+  "emoji": "🚘",
+  "description": "oncoming automobile",
+  "aliases": ["oncoming_automobile"],
+  "tags": []
+}, {
+  "emoji": "🚗",
+  "description": "automobile",
+  "aliases": ["car", "red_car"],
+  "tags": []
+}, {
+  "emoji": "🚕",
+  "description": "taxi",
+  "aliases": ["taxi"],
+  "tags": []
+}, {
+  "emoji": "🚖",
+  "description": "oncoming taxi",
+  "aliases": ["oncoming_taxi"],
+  "tags": []
+}, {
+  "emoji": "🚛",
+  "description": "articulated lorry",
+  "aliases": ["articulated_lorry"],
+  "tags": []
+}, {
+  "emoji": "🚚",
+  "description": "delivery truck",
+  "aliases": ["truck"],
+  "tags": []
+}, {
+  "emoji": "🚨",
+  "description": "police cars revolving light",
+  "aliases": ["rotating_light"],
+  "tags": ["911", "emergency"]
+}, {
+  "emoji": "🚓",
+  "description": "police car",
+  "aliases": ["police_car"],
+  "tags": []
+}, {
+  "emoji": "🚔",
+  "description": "oncoming police car",
+  "aliases": ["oncoming_police_car"],
+  "tags": []
+}, {
+  "emoji": "🚒",
+  "description": "fire engine",
+  "aliases": ["fire_engine"],
+  "tags": []
+}, {
+  "emoji": "🚑",
+  "description": "ambulan
```

Comment:
How come `_collection`? I can't find any references to it.

Reply:
Afaik it's the only way I know of to insert documents on the client only (and yes, it's a hack). But meteor seems to clear the collection when a hot code reload happens, so we need to repopulate it. I just can't find a hook for that. There is Meteor._onMigrate but that runs before the reload takes place. I want to know when a reload just happened.

Reply:
Gotcha. Looks good to me then!

Too bad about no post-action on a reload :/ Otherwise we could've stored away the collection's content just before a reload, and then re-seeding.

Reply:
https://github.com/meteor/meteor/blob/master/packages/reload/reload.js#L122

This will probably work.

Reply:
Ackschhtually, it looks like it does work as it is. I think I ran into another error that prevented the emojis from loading.

Reply:
Nice!